app/src/main/java/org/piramalswasthya/cho/ui/commons/lab_technician/LabTechnicianFormViewModel.kt

```
@@ -109,6 +155,17 @@ class LabTechnicianFormViewModel @Inject constructor(
         _boolCall.value = false
     }
 
+    fun getVisitReasonByBenFlowID(beneficiaryID: Long) {
+        viewModelScope.launch {
+            try {
+                val benFlowList = benFlowRepo.getBenFlowByBenFlowID(beneficiaryID)
+                _benFlows.value = benFlowList
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)